src/index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link href="https://fonts.googleapis.com/css?family=Google+Sans:100,300,400,700" rel="stylesheet">
+    <link href="assets/css/index.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+    <header>
+        <div class="header_title">
+            <div class="top-title wrapper">
+                <img alt="areteos coin logo" class="header_img" src="assets/img/areteos-logo-full.png" />
+            </div>
+        </div>
+        <div class="header_bar">
+            <div class="header wrapper">
+                <div class="title">Areteos Coin</div>
+                <div class="sub-title">Your gateway to global investment opportunities</div>
+                <p>Areteos Coin is a stablecoin that’s backed by the Global Group,
+                    giving you preferential access to
+                    a world of investment opportunities.</p>
+                <button type="button" class="header_button">Purchase Areteos Coin</button>
```

Comment:
does this button links to other pages? :)

Reply:
for now its not. maybe later